CHANGELOG.md

```diff
@@ -26,10 +26,12 @@ All notable changes to this project will be documented in this file.
 - Document that Spark Connect doesn't integrate with the history server ([#559])
 - test: Bump to Vector `0.46.1` ([#565]).
 - Use versioned common structs ([#572]).
+- Change the label `app.kubernetes.io/name` for Spark history and connect objects to use `spark-history` and `spark-connect` instead of `spark-k8s` ([#573]).
```

Comment:
I guess we need to cakk out as breaking change, users need to delete their Statefulsets, as the labels are used in the volumeCliamTemplate IIRC

Reply:
True. https://github.com/stackabletech/spark-k8s-operator/pull/573/commits/db8ed80baf4a8d0bb800b86c1cadc3bfe6f3d613